examples/booking_app/lib/feature_hotel/search/blocs/hotel_list_bloc_extensions.dart

```diff
@@ -13,14 +13,14 @@ extension _ReloadDataFetcher on Stream<_ReloadData> {
       switchMap(
         (reloadData) {
           if (reloadData.reset) {
-            _paginatedList.value!.reset(hard: reloadData.fullReset);
+            _paginatedList.value.reset(hard: reloadData.fullReset);
```

Comment:
@DDavidPrime, now the [Behaviour.subject](https://pub.dev/documentation/rxdart/latest/rx/BehaviorSubject/value.html) is not nullable, great :)